core/server/data/migrations/versions/4.12/02-fix-member-statuses.js

```diff
@@ -5,10 +5,21 @@ module.exports = createTransactionalMigration(
     async function up(knex) {
         logging.info('Updating members with no products to "free"');
 
-        await knex('members')
+        const freeMembers = await knex('members')
+            .select('members.id')
             .leftJoin('members_products', 'members.id', 'members_products.member_id')
-            .update('status', 'free')
```

Comment:
SQLite3 doesn't support updating with a `JOIN` so I've refactored this to use `WHERE IN`